lms/js_config_types.py

```diff
@@ -118,6 +118,9 @@ class DashboardRoutes(TypedDict):
     students: str
     """Paginated endpoint to fetch students"""
 
+    assignment_grades_sync: str
```

Comment:
I reckon this name could work for both the:

POST - existing one, schedule a new sync
GET - check if a sync is in progress, track postgres